daemon/algod/api/algod.oas3.yml

```diff
@@ -630,6 +630,30 @@
         },
         "description": "Transaction ID of the submission."
       },
+      "SimulationResponse": {
+        "content": {
+          "application/json": {
+            "schema": {
+              "properties": {
+                "failure-message": {
+                  "description": "\\[fm\\] Failure message, if the transaction would have failed during a live broadcast.",
+                  "type": "string"
+                },
+                "missing-signatures": {
+                  "description": "\\[ms\\] Whether any transactions would have failed during a live broadcast because they were missing signatures.",
+                  "type": "boolean"
+                }
```

Comment:
This response will change significantly in follow-up PRs, so I suggest not giving its current form too much thought